community/cypher/cypher-compiler-3.2/src/main/scala/org/neo4j/cypher/internal/compiler/v3_2/commands/expressions/ShortestPathExpression.scala

```diff
@@ -176,7 +174,5 @@ case class ShortestPathExpression(shortestPathPattern: ShortestPath, predicates:
           }
       }
 
-  private def doesNotDependOnFullPath(predicate: Predicate): Boolean = {
-    (predicate.symbolTableDependencies intersect pathVariables).isEmpty
-  }
+  private def doesNotDependOnFullPath(predicate: Predicate): Boolean = false // TODO: This is not correct!
```

Comment:
We could extend the `Predicate` class to contain a set of dependencies, but perhaps that's what we aimed to get rid of with this commit?